registry.go

```diff
@@ -30,54 +30,53 @@ import (
 // performing operations on an entity as well as helper methods for accessing
 // type data so that reflection can be minimized.
 type RegisteredEntity struct {
-	scope  string
-	prefix string
-	table  *Table
-	info   *EntityInfo
-	typ    reflect.Type // optimization to avoid doing repetitive reflect.TypeOf
+	scope     string
```

Comment:
this change seems completely unrelated to adding TTLs. Can you please move it to a separate PR?